src/components/SearchBox/index.js

```diff
@@ -1,36 +1,59 @@
 import React, { useState } from "react";
 import { Button, Form, Col, Row, Container } from "react-bootstrap";
-import { useDispatch } from "react-redux";
-import { getBooks } from "../../store/search/actions";
-
-export default function SearchBox() {
-  const dispatch = useDispatch();
 
+export default function SearchBox(props) {
   const [language, setLanguage] = useState("all");
   const [distance, setDistance] = useState("all");
+  const [title, setTitle] = useState("all");
+  const [latitude, setLatitude] = useState(0);
+  const [longitude, setLongitude] = useState(0);
+
+  const clickHandler = event => {
+    event.preventDefault();
+    console.log(
+      "Search is Clicked!",
+      title,
+      "title",
+      language,
+      "language",
+      distance,
+      "distance",
+      latitude,
+      "latitude",
+      longitude,
+      "longitude"
+    );
+    props.getBooks(title, language, distance, latitude, longitude);
+    console.log("PROPS", title, language, distance, latitude, longitude);
+  };
 
-  const clickHandler = () => {
-    dispatch(getBooks());
+  const showPosition = position => {
```

Comment:
This is hard to understand, it looks like there is no need for show position, state updates can be done inside the get coordinates function.